src/ui/components/Timeline/FocusModePopout.tsx

```diff
@@ -43,16 +35,14 @@ export default function FocusModePopout({
   const discardPendingChanges = async (isImplicit: boolean) => {
     hideModal();
 
-    if (updateFocusWindowThrottled.hasPending()) {
-      updateFocusWindowThrottled.cancel();
-    }
-
     const currentFocusWindow = replayClient.getCurrentFocusWindow();
     assert(currentFocusWindow);
     dispatch(
       setFocusWindow({ begin: currentFocusWindow.begin.time, end: currentFocusWindow.end.time })
     );
 
+    dispatch(setHoverTime(null, false));
```

Comment:
nit: We could use a comment explaining why we do this. (Your explanation in the Linear issue makes sense but glancing at the code, it's maybe not obvious.)